src/js/shared/constants.js

```diff
@@ -1,11 +1,15 @@
 export const MiscURLs = {
   ABOUT_US_URL: "https://www.criticalstart.com/company/",
+  CRITICALSTART_URL: "https://www.criticalstart.com",
   EXTENSION_HOME_URL:
     "https://github.com/AdvancedThreatAnalytics/threat-analytics-search",
   INSTALLED_URL:
     "https://www.criticalstart.com/threat-analytics-chrome-plugin/",
   ISSUES_URL:
     "https://github.com/AdvancedThreatAnalytics/threat-analytics-search/issues",
+  TAS_URL: "https://github.com/AdvancedThreatAnalytics/threat-analytics-search",
+  TAS_RELEASES_URL:
```

Comment:
Please change to jus `RELEASES_URL` to match the naming convention of the other constants.